.github/workflows/contributor.yaml

```diff
@@ -30,7 +30,20 @@ jobs:
       - name: Author of comment has insufficient permissions
         if: ${{ !contains(fromJson('["OWNER", "COLLABORATOR", "MEMBER"]'), github.event.comment.author_association) }}
         run: |
-          echo ${{ github.event.comment.author_association }}
+          AUTHOR_ASSOCIATION=${{ github.event.comment.author_association }}
+          echo "${AUTHOR_ASSOCIATION}"
+          if [[ "${AUTHOR_ASSOCIATION}" == "CONTRIBUTOR" ]]; then
```

Comment:
It could be `NONE` as well ([see here](https://github.com/orgs/community/discussions/18690#discussioncomment-4232389)). Imagine a case when a user who hasn't contributed to the repository and has been added as a `MEMBER`.

Reply:
Fair enough, but I would put someone who has never contributed to the code base in the same category as an "outside" contributor. We can also remove the `if` completely and always show the instructions.